resources/fetching/fetchers/gcp/assets_fetcher.go

```diff
@@ -40,19 +44,23 @@ type GcpAsset struct {
 	Asset *assetpb.Asset `json:"asset,omitempty"`
 }
 
-var GcpAssetTypes = map[string]map[string][]string{
+// map of types to asset types.
+// sub-type is derived from asset type by using the first and last segments of the asset type name
+// example: gcp-cloudkms-cryptokey
+var GcpAssetTypes = map[string][]string{
```

Comment:
reason for this change is that we came up with sub types names ourselves which we derived manually from the asset types. this change makes the map compact and clean.